src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 50px;
+}
+
+.link {
+  color: black;
+  text-decoration: none;
+  margin-left: 20px;
+  list-style: hover;
+  display: flex;
+  height: 60px;
+  line-height: 60px;
+}
+
+.wraper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  white-space: nowrap;
+}
+
+html {
```

Comment:
please follow the order of writing styles :tag selectors and styles for them should go first, then class selectors in the order in which these elements are in the code